integration.md

```diff
@@ -210,6 +213,42 @@ every 1.day do
 end
 ```
 
+## Preferences
+After is user is registered with Klarna, you might want to allow them to view and possibly change their current payment preferences.
+Since the preferences form displays sensitive data about the user without prior authenticion, rendering the flow is a 2-step process: a preferences session is first securly created, and then the preferenecs data is displayed against the session identifier.
+
+### Acquiring a preferences session id
+Once a user has chosen to view their prefernces (by clicking a designated button, or in any other form you see fit), your backend server should acquire a preferences session identifier. This is done by calling the following method:
+
+    URL:
+      production: https://ondemand.klarna.com/web
+      playground: https://ondemand-dg.playground.klarna.com/web
+    Action: preferneces
+    Method: POST
+    Params:
+      api_key: your api key. Required
+      api_secret: your api secret. Required
+      user_token: the requesting user's token. Required
+    Success Response:
+      Code: 201
+      Content: { session_id: <session_id> }
+    Error Response:
+      Code: 402
+      Content: { error : "credentials_mismatch" }
+
+### Placing a preferences form on your page
+Assuming you've already included the SDK itself on your page, place the preferences form in the page. This form will allow the user to view and change their preferences:
```

Comment:
I think saying something like the following will be clearer

Place a Klarna form in your rendered page. This form will open a Klarna frame that will allow...

WDYT?